hummingbot/wallet/ethereum/mock_wallet.pyx

```diff
@@ -77,26 +77,26 @@ cdef class MockWallet(WalletBase):
         except Exception:
             get_logger().error("Error sending transaction to Ethereum network.", exc_info=True)
 
-    def get_all_balances(self) -> Dict[str, float]:
+    def get_all_balances(self) -> Dict[str, Decimal]:
         retval = {"ETH": self.c_get_balance("ETH")}
         for asset_name in self._erc20_contracts.keys():
             retval[asset_name] = self.c_get_balance(asset_name)
         return retval
 
-    cdef double c_get_balance(self, str asset_name) except? -1:
+    cdef object c_get_balance(self, str asset_name):
         if asset_name == "ETH":
-            return self._w3.eth.getBalance(self.address) * 1e-18
+            return Decimal(self._w3.eth.getBalance(self.address)) / 10 ** 18
```

Comment:
While this is technically valid because `10 ** 18` is an `int`.. it's not a good convention to follow because the same syntax can easily yield floats. Let's make it clear by doing `Decimal(getBalance()) * Decimal("1e-18")` instead.